src/module3/provided/task-3-export-csv-to-txt.ts

```diff
@@ -0,0 +1,81 @@
+import * as fs from 'fs';
+import { pipeline, Transform } from 'stream';
+import readline from 'readline';
+
+/**
+ * Exports a CSV file to a TXT file.
+ * CSV format:
+ *  Book;Author;Amount;Price
+ *  The Compound Effect;Darren Hardy;5;9,48
+ * TXT format: {"book":"The Compound Effect","author":"Darren Hardy","price":9.48}
+ * @param {string} csvPath - The path to the source CSV file.
+ * @param {string} txtPath - The path to the destination TXT file.
+ * @returns {Promise<boolean>} - A promise that resolves to true when export is done.
+ */
+export const exportCsvToTxt = (csvPath: string, txtPath: string): Promise<boolean> => {
+  // implementation here
+  console.log(`csvPath = ${csvPath}, txtPath = ${txtPath}`);
+
+  // create Promise that resolves to true when export is done
+  let exportStatusResolver: (arg0: boolean) => void;
+  const exportStatus = new Promise<boolean>((resolve) => {
+    exportStatusResolver = resolve;
+  });
+
+  
+  // create a readable stream from the CSV file
+  const readStream = fs.createReadStream(csvPath);
+
+  // create Stream that will read the CSV file line by line
+  const readLineStream = readline.createInterface({
+    input: readStream,
+    crlfDelay: Infinity
+  });
+
+
+  // create a writable stream to the TXT file
+  const writeStream = fs.createWriteStream(txtPath, { encoding: 'utf8' });
+  
+  // create a transform stream to convert CSV to TXT
+  const transformStream = new Transform({
```

Comment:
Missed this AC: "convert the data to JSON format using the [csvtojson](https://www.npmjs.com/package/csvtojson) package"

Reply:
uff
Its in json format though 😅